### ISSUE TITLE
Multicast fails for just under half a minute every minute

### ISSUE THREAD

**Issue by key-networks** [zerotier/ZeroTierOne#1178]
**Describe the bug**
Running a multicast test with iperf between two nodes in a ZT network results in no traffic for just under half a minute every minute.  Running the same test as unicast produces no such failure.

**To Reproduce**
1. Set up 2 Linux ZT nodes - can be on the same network.
2. Set route for multicast via ZT interface on both sides, e.g.:
`ip r add 224.0.0.0/4 dev ztc2bhzkqs`
3. Run multicast sender on one node:
`iperf -c 239.255.33.33 -p 7777 -u -T32 -i1 -b1k -t6000`
4. Run multicast listener on the other node:
`iperf -s -u -B 239.255.33.33 -p 7777 -i1`
5. See traffic on the listener die for 28 seconds or so every minute.
6. Note that a tcpdump on the ZT interface at the sender proves that it is sending the whole time.

**Expected behavior**
Multicast traffic should continue as long as the sender is sending.

**Screenshots**
$ iperf -s -u -B 239.255.33.33 -p 7777 -i1
Server listening on UDP port 7777
Binding to local address 239.255.33.33
Joining multicast group  239.255.33.33
Receiving 1470 byte datagrams
UDP buffer size:  208 KByte (default)
[  3] local 239.255.33.33 port 7777 connected with 10.14.104.36 port 58738
[ ID] Interval       Transfer     Bandwidth        Jitter   Lost/Total Datagrams
[  3]  0.0- 1.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3]  1.0- 2.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3]  2.0- 3.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3]  3.0- 4.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3]  4.0- 5.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3]  5.0- 6.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3]  6.0- 7.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3]  7.0- 8.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3]  8.0- 9.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3]  9.0-10.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 10.0-11.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 11.0-12.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 12.0-13.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 13.0-14.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 14.0-15.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 15.0-16.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 16.0-17.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 17.0-18.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 18.0-19.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 19.0-20.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 20.0-21.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 21.0-22.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 22.0-23.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 23.0-24.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 24.0-25.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 25.0-26.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 26.0-27.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 27.0-28.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 28.0-29.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 29.0-30.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 30.0-31.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 31.0-32.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 32.0-33.0 sec  0.00 Bytes  0.00 bits/sec   0.000 ms    0/    0 (-nan%)
[  3] 33.0-34.0 sec  1.44 KBytes  11.8 Kbits/sec   0.000 ms   67/   68 (99%)
[  3] 34.0-35.0 sec  1.44 KBytes  11.8 Kbits/sec   0.022 ms    0/    1 (0%)
[  3] 35.0-36.0 sec  1.44 KBytes  11.8 Kbits/sec   0.022 ms    0/    1 (0%)
[  3] 36.0-37.0 sec  1.44 KBytes  11.8 Kbits/sec   0.022 ms    0/    1 (0%)
[  3] 37.0-38.0 sec  1.44 KBytes  11.8 Kbits/sec   0.029 ms    0/    1 (0%)
[  3] 38.0-39.0 sec  1.44 KBytes  11.8 Kbits/sec   0.070 ms    0/    1 (0%)
[  3] 39.0-40.0 sec  1.44 KBytes  11.8 Kbits/sec   0.112 ms    0/    1 (0%)
[  3] 40.0-41.0 sec  1.44 KBytes  11.8 Kbits/sec   0.108 ms    0/    1 (0%)
[  3] 41.0-42.0 sec  1.44 KBytes  11.8 Kbits/sec   0.104 ms    0/    1 (0%)
[  3] 42.0-43.0 sec  1.44 KBytes  11.8 Kbits/sec   0.107 ms    0/    1 (0%)
[  3] 43.0-44.0 sec  1.44 KBytes  11.8 Kbits/sec   0.114 ms    0/    1 (0%)
[  3] 44.0-45.0 sec  1.44 KBytes  11.8 Kbits/sec   0.108 ms    0/    1 (0%)
[  3] 45.0-46.0 sec  1.44 KBytes  11.8 Kbits/sec   0.108 ms    0/    1 (0%)
[  3] 46.0-47.0 sec  1.44 KBytes  11.8 Kbits/sec   0.104 ms    0/    1 (0%)
[  3] 47.0-48.0 sec  1.44 KBytes  11.8 Kbits/sec   0.098 ms    0/    1 (0%)
[  3] 48.0-49.0 sec  1.44 KBytes  11.8 Kbits/sec   0.093 ms    0/    1 (0%)
[  3] 49.0-50.0 sec  1.44 KBytes  11.8 Kbits/sec   0.089 ms    0/    1 (0%)
[  3] 50.0-51.0 sec  1.44 KBytes  11.8 Kbits/sec   0.087 ms    0/    1 (0%)
[  3] 51.0-52.0 sec  1.44 KBytes  11.8 Kbits/sec   0.101 ms    0/    1 (0%)
[  3] 52.0-53.0 sec  1.44 KBytes  11.8 Kbits/sec   0.146 ms    0/    1 (0%)
[  3] 53.0-54.0 sec  1.44 KBytes  11.8 Kbits/sec   0.207 ms    0/    1 (0%)
[  3] 54.0-55.0 sec  1.44 KBytes  11.8 Kbits/sec   0.213 ms    0/    1 (0%)
[  3] 55.0-56.0 sec  1.44 KBytes  11.8 Kbits/sec   0.217 ms    0/    1 (0%)
[  3] 56.0-57.0 sec  1.44 KBytes  11.8 Kbits/sec   0.260 ms    0/    1 (0%)
[  3] 57.0-58.0 sec  1.44 KBytes  11.8 Kbits/sec   0.296 ms    0/    1 (0%)
[  3] 58.0-59.0 sec  1.44 KBytes  11.8 Kbits/sec   0.279 ms    0/    1 (0%)
[  3] 59.0-60.0 sec  1.44 KBytes  11.8 Kbits/sec   0.504 ms    0/    1 (0%)
[  3] 60.0-61.0 sec  1.44 KBytes  11.8 Kbits/sec   0.713 ms    0/    1 (0%)
[  3] 61.0-62.0 sec  1.44 KBytes  11.8 Kbits/sec   0.669 ms    0/    1 (0%)
[  3] 62.0-63.0 sec  1.44 KBytes  11.8 Kbits/sec   0.634 ms    0/    1 (0%)
[  3] 63.0-64.0 sec  1.44 KBytes  11.8 Kbits/sec   1.023 ms    0/    1 (0%)
[  3] 64.0-65.0 sec  1.44 KBytes  11.8 Kbits/sec   1.374 ms    0/    1 (0%)
[  3] 65.0-66.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 66.0-67.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 67.0-68.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 68.0-69.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 69.0-70.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 70.0-71.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 71.0-72.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 72.0-73.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 73.0-74.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 74.0-75.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 75.0-76.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 76.0-77.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 77.0-78.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 78.0-79.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 79.0-80.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 80.0-81.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 81.0-82.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 82.0-83.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 83.0-84.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 84.0-85.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 85.0-86.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 86.0-87.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 87.0-88.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 88.0-89.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 89.0-90.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 90.0-91.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 91.0-92.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 92.0-93.0 sec  0.00 Bytes  0.00 bits/sec   1.374 ms    0/    0 (-nan%)
[  3] 93.0-94.0 sec  1.44 KBytes  11.8 Kbits/sec   1.310 ms   28/   29 (97%)
[  3] 94.0-95.0 sec  1.44 KBytes  11.8 Kbits/sec   1.244 ms    0/    1 (0%)
[  3] 95.0-96.0 sec  1.44 KBytes  11.8 Kbits/sec   1.167 ms    0/    1 (0%)
[  3] 96.0-97.0 sec  1.44 KBytes  11.8 Kbits/sec   1.096 ms    0/    1 (0%)
[  3] 97.0-98.0 sec  1.44 KBytes  11.8 Kbits/sec   1.029 ms    0/    1 (0%)
[  3] 98.0-99.0 sec  1.44 KBytes  11.8 Kbits/sec   0.969 ms    0/    1 (0%)
[  3] 99.0-100.0 sec  1.44 KBytes  11.8 Kbits/sec   0.908 ms    0/    1 (0%)
[  3] 100.0-101.0 sec  1.44 KBytes  11.8 Kbits/sec   0.852 ms    0/    1 (0%)
[  3] 101.0-102.0 sec  1.44 KBytes  11.8 Kbits/sec   0.804 ms    0/    1 (0%)
[  3] 102.0-103.0 sec  1.44 KBytes  11.8 Kbits/sec   0.759 ms    0/    1 (0%)
[  3] 103.0-104.0 sec  1.44 KBytes  11.8 Kbits/sec   0.716 ms    0/    1 (0%)
[  3] 104.0-105.0 sec  1.44 KBytes  11.8 Kbits/sec   0.676 ms    0/    1 (0%)
[  3] 105.0-106.0 sec  1.44 KBytes  11.8 Kbits/sec   0.636 ms    0/    1 (0%)
[  3] 106.0-107.0 sec  1.44 KBytes  11.8 Kbits/sec   0.646 ms    0/    1 (0%)
[  3] 107.0-108.0 sec  1.44 KBytes  11.8 Kbits/sec   0.645 ms    0/    1 (0%)
[  3] 108.0-109.0 sec  1.44 KBytes  11.8 Kbits/sec   1.754 ms    0/    1 (0%)
[  3] 109.0-110.0 sec  1.44 KBytes  11.8 Kbits/sec   2.779 ms    0/    1 (0%)
[  3] 110.0-111.0 sec  1.44 KBytes  11.8 Kbits/sec   2.614 ms    0/    1 (0%)
[  3] 111.0-112.0 sec  1.44 KBytes  11.8 Kbits/sec   2.451 ms    0/    1 (0%)
[  3] 112.0-113.0 sec  1.44 KBytes  11.8 Kbits/sec   2.304 ms    0/    1 (0%)
[  3] 113.0-114.0 sec  1.44 KBytes  11.8 Kbits/sec   2.184 ms    0/    1 (0%)
[  3] 114.0-115.0 sec  1.44 KBytes  11.8 Kbits/sec   2.069 ms    0/    1 (0%)
[  3] 115.0-116.0 sec  1.44 KBytes  11.8 Kbits/sec   1.943 ms    0/    1 (0%)
[  3] 116.0-117.0 sec  1.44 KBytes  11.8 Kbits/sec   1.827 ms    0/    1 (0%)
[  3] 117.0-118.0 sec  1.44 KBytes  11.8 Kbits/sec   1.721 ms    0/    1 (0%)
[  3] 118.0-119.0 sec  1.44 KBytes  11.8 Kbits/sec   1.620 ms    0/    1 (0%)
[  3] 119.0-120.0 sec  1.44 KBytes  11.8 Kbits/sec   1.520 ms    0/    1 (0%)
[  3] 120.0-121.0 sec  0.00 Bytes  0.00 bits/sec   1.520 ms    0/    0 (-nan%)
[  3] 121.0-122.0 sec  2.87 KBytes  23.5 Kbits/sec   4.149 ms    0/    2 (0%)
[  3] 122.0-123.0 sec  1.44 KBytes  11.8 Kbits/sec   3.898 ms    0/    1 (0%)
[  3] 123.0-124.0 sec  1.44 KBytes  11.8 Kbits/sec   3.660 ms    0/    1 (0%)
[  3] 124.0-125.0 sec  1.44 KBytes  11.8 Kbits/sec   3.437 ms    0/    1 (0%)
[  3] 125.0-126.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 126.0-127.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 127.0-128.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 128.0-129.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 129.0-130.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 130.0-131.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 131.0-132.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 132.0-133.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 133.0-134.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 134.0-135.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 135.0-136.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 136.0-137.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 137.0-138.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 138.0-139.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 139.0-140.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 140.0-141.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 141.0-142.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 142.0-143.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 143.0-144.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 144.0-145.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 145.0-146.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 146.0-147.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 147.0-148.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 148.0-149.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 149.0-150.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 150.0-151.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 151.0-152.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 152.0-153.0 sec  0.00 Bytes  0.00 bits/sec   3.437 ms    0/    0 (-nan%)
[  3] 153.0-154.0 sec  1.44 KBytes  11.8 Kbits/sec   3.286 ms   28/   29 (97%)
[  3] 154.0-155.0 sec  1.44 KBytes  11.8 Kbits/sec   3.106 ms    0/    1 (0%)
[  3] 155.0-156.0 sec  1.44 KBytes  11.8 Kbits/sec   2.921 ms    0/    1 (0%)
[  3] 156.0-157.0 sec  1.44 KBytes  11.8 Kbits/sec   2.743 ms    0/    1 (0%)
^CWaiting for server threads to complete. Interrupt again to force quit.
[  3] 157.0-158.0 sec  1.44 KBytes  11.8 Kbits/sec   2.575 ms    0/    1 (0%)
[  3] 158.0-159.0 sec  1.44 KBytes  11.8 Kbits/sec   2.420 ms    0/    1 (0%)

**Desktop (please complete the following information):**
 - OS: Linux
 - OS/Distribution Version: Ubuntu 16.04.6 LTS and Fedora 31
 - ZeroTier Version: 1.4.6
 - Hardware: DELL Vostro i5, Asus eee PC, a few others tested as well

**Additional context**
- Running my own controller on 1.4.6
- ZeroTier Network Configuration: 2 nodes on network, default config
- Router Config: on the same LAN or on different networks - same issue
- Firewall Config (try turning the firewall off): yes
- General Network Environment: Home and work

**Comment by key-networks** [comment]
This appears to be a regression introduced in 1.4.4.  I have tested these and they passed (one node on Fedora 31 and the other on Ubuntu 16.04):
- 1.2.0
- 1.2.8
- 1.4.0
- 1.4.2

The following fail:
- 1.4.4
- 1.4.6

**Comment by laduke** [comment]
I'm seeing the same thing. 

```
# ping -b 100.64.64.255
WARNING: pinging broadcast address
PING 100.64.64.255 (100.64.64.255) 56(84) bytes of data.
64 bytes from 100.64.64.73: icmp_seq=2 ttl=64 time=87.9 ms
64 bytes from 100.64.64.73: icmp_seq=3 ttl=64 time=139 ms
64 bytes from 100.64.64.73: icmp_seq=4 ttl=64 time=88.2 ms
64 bytes from 100.64.64.73: icmp_seq=5 ttl=64 time=156 ms
64 bytes from 100.64.64.73: icmp_seq=6 ttl=64 time=123 ms
64 bytes from 100.64.64.73: icmp_seq=7 ttl=64 time=173 ms
64 bytes from 100.64.64.73: icmp_seq=8 ttl=64 time=95.2 ms
64 bytes from 100.64.64.73: icmp_seq=9 ttl=64 time=283 ms
64 bytes from 100.64.64.73: icmp_seq=10 ttl=64 time=87.5 ms
64 bytes from 100.64.64.73: icmp_seq=11 ttl=64 time=102 ms
64 bytes from 100.64.64.73: icmp_seq=12 ttl=64 time=92.8 ms
64 bytes from 100.64.64.73: icmp_seq=13 ttl=64 time=92.7 ms
64 bytes from 100.64.64.73: icmp_seq=14 ttl=64 time=164 ms
64 bytes from 100.64.64.73: icmp_seq=15 ttl=64 time=135 ms
64 bytes from 100.64.64.73: icmp_seq=16 ttl=64 time=87.8 ms
64 bytes from 100.64.64.73: icmp_seq=17 ttl=64 time=89.2 ms
64 bytes from 100.64.64.73: icmp_seq=18 ttl=64 time=104 ms
64 bytes from 100.64.64.73: icmp_seq=19 ttl=64 time=96.4 ms
64 bytes from 100.64.64.73: icmp_seq=20 ttl=64 time=86.9 ms
64 bytes from 100.64.64.73: icmp_seq=21 ttl=64 time=102 ms
64 bytes from 100.64.64.73: icmp_seq=22 ttl=64 time=88.1 ms
~30 seconds pass with no output
64 bytes from 100.64.64.73: icmp_seq=59 ttl=64 time=244 ms
64 bytes from 100.64.64.73: icmp_seq=60 ttl=64 time=97.3 ms
64 bytes from 100.64.64.73: icmp_seq=61 ttl=64 time=88.5 ms
64 bytes from 100.64.64.73: icmp_seq=62 ttl=64 time=85.1 ms
64 bytes from 100.64.64.73: icmp_seq=63 ttl=64 time=99.7 ms
...
```

This happens if the pinger is 1.4.4 or 1.4.6.

1.4.2 works as intended.

**Comment by laduke** [comment]
👋 This got fixed in 1.6 right?
